src/cdk/drag-drop/drag.ts

```diff
@@ -590,6 +605,30 @@ export class CdkDrag<T = any> implements OnDestroy {
 
     this._placeholder = this._placeholderRef = null!;
   }
+
+  /** Updates the current drag delta, based on the user's current pointer position on the page. */
+  private _updateDelta(pointerPositionOnPage: Point) {
```

Comment:
`_updatePointerDirectionDelta`?

`pointerPositionOnPage` -> `pointerPosition`? (isn't it always on the page?)

I'm also wondering if would make sense to refactor most of the pointer-related code into a separate class like `PointerState`; this file is getting pretty hefty, and it would be nice to separate the logic of the drag/drop behavior from the Angular-specific stuff.

Reply:
WRT the parameter naming, I wanted to make the distinction that it's relative to the page, rather than within the element, which we also keep track of inside the `drag.ts`.

Regarding refactoring, I agree that we should do it before we end up in a situation like `mat-select`. I've added it to my list.